contentcuration/contentcuration/frontend/shared/views/GlobalSnackbar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+
+  <v-snackbar
+    v-if="snackbarIsVisible"
+    :key="key"
+    :timeout="snackbarOptions.duration"
+    bottom
+    :value="snackbarIsVisible"
```

Comment:
If hideCallback isn't called, would it make sense to have an @input listener to clear the snackbar? Something like
```
visibilityToggled(visible) {
    if(!visible) {
        this.$store.commit('CORE_CLEAR_SNACKBAR');
    }
}
```

Reply:
Done :+1: